site/content/docs/tutorials/local-concierge-and-supervisor-demo.md

```diff
@@ -0,0 +1,682 @@
+    organizations:
+      - Project Pinniped
+    organizationalUnits:
+      - Demo
+  secretName: demo-ca-secret
+  privateKey:
+    algorithm: ECDSA
+    size: 256
+  issuerRef:
+    group: cert-manager.io
+    kind: Issuer
+    name: selfsigned-issuer
+---
+# Create an issuer that will sign certs with our self-signed CA.
+apiVersion: cert-manager.io/v1
+kind: Issuer
+metadata:
+  name: demo-ca-issuer
+  namespace: pinniped-supervisor
+spec:
+  ca:
+    secretName: demo-ca-secret
+---
+# Finally, create serving certs using our CA.
+# This generates a Secret which is the only output of
+# cert-manager that the Supervisor needs to read.
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  name: supervisor-tls-cert-request
+  namespace: pinniped-supervisor
+spec:
+  secretName: supervisor-tls-cert
+  issuerRef:
+    name: demo-ca-issuer
+  dnsNames:
+  - pinniped-supervisor.pinniped-supervisor.svc.cluster.local
+EOF
+```
+
+Wait for the Secret to get created. This may take a few seconds.
+Then read the CA's public key from the Secret and save it locally for later use.
+
+```sh
+kubectl get secret supervisor-tls-cert \
+  --namespace pinniped-supervisor \
+  -o jsonpath="{.data['ca\.crt']}" | base64 -d > /tmp/supervisor-ca.crt
+```
+
+### Configure an Identity Provider in the Pinniped Supervisor
+
+For this tutorial, we will use GitHub as an identity provider.
+If you'd rather try another identity provider type, the steps would be roughly the same.
+See the Pinniped documentation for other supported identity provider types.
+
+You will need to give the Supervisor permission to help you log in to GitHub. Pinniped will redirect you to
+github.com in your web browser for login. Pinniped will never see your GitHub credentials. It will only be able to read
+your basic profile information (e.g. your username) and some information about your organization and team memberships.
+
+1. Log in to GitHub. If you do not have a GitHub account, you can sign up for free.
+2. Click on your profile icon in the top-right corner, and choose
+   Settings -> Developer Settings -> OAuth Apps -> New OAuth App. Fill out the form:
+   - Application Name: `Pinniped demo`, or any other name that you prefer.
+   - Homepage URL: `https://pinniped.dev`, or any other URL that you prefer.
+   - Authorization callback URL: `https://pinniped-supervisor.pinniped-supervisor.svc.cluster.local/demo-issuer/callback`.
+     You must use exactly this value for this tutorial.
+   - Enable device flow: leave this box unchecked
+2. Click "Register Application"
+3. Click "Generate New Client Secret"
+4. Copy the client secret for use in the next step. It will not be shown to you again.
+5. Also copy the client ID for use in the next step.
+
+```sh
+# Set the client ID and client secret as env vars for use in the next step.
+export GITHUB_APP_CLIENT_ID=<paste client ID here>
+export GITHUB_APP_CLIENT_SECRET=<paste client secret here>
+```
+
+Configure the identity provider.
```

Comment:
This example uses the github identity provider. Suggest adding links to existing docs to use other IDPs.

Reply:
Thanks! I force pushed an update with your suggestion.